compute/instances/api-cli/managing-instance-snapshot-via-cli.mdx

```diff
@@ -0,0 +1,91 @@
+---
+meta:
+  title: Managing Instance snapshots with the CLI (v2)
+  description: This page explains how to manage Instance snapshots with the CLI (v2)
+content:
+  h1: Managing Instance snapshots with the CLI (v2)
+  paragraph: This page explains how to manage Instance snapshots with the CLI (v2)
+dates:
+  validation: 2022-09-21
+  posted: 2022-09-24
+---
+
+You can use the Scaleway CLI to create, export and import snapshots of your Instances. A snapshot is a QCOW2 image of your Instances volumes and stored in a Scaleway Object Storage bucket. It can be imported into any Availability Zone, as long as it is in the same region as the bucket in which it hosted.
```

Comment:
```suggestion
You can use the Scaleway CLI to create, export and import snapshots of your Instances. A snapshot is a QCOW2 image of your Instance's volumes, stored in a Scaleway Object Storage bucket. It can be imported into any Availability Zone, as long as it is in the same region as the bucket in which it is hosted.
```